packages/dx-react-scheduler-demos/src/demo-sources/scheduler-localization/material-ui/locale.jsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  AllDayPanel,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+import { appointments } from '../../../demo-data/appointments';
+
+const allDayLocalizationMessages = {
+  'fr-FR': {
+    allDay: "Autour de l'horloge",
+  },
+  'de-GR': {
+    allDay: 'Rund um die Uhr',
+  },
+  'en-US': {
+    allDay: 'All Day',
+  },
+};
+
+const getAllDayMessages = locale => allDayLocalizationMessages[locale];
+
+const styles = theme => ({
+  container: {
+    display: 'flex',
+    marginBottom: theme.spacing.unit * 2,
+    justifyContent: 'flex-end',
+  },
+  text: {
+    ...theme.typography.h6,
+    marginRight: theme.spacing.unit * 2,
+  },
+});
+
+const LocaleSwitcher = withStyles(styles, { name: 'LocaleSwitcher' })(
+  ({ onLocaleChange, currentLocale, classes }) => (
+    <div className={classes.container}>
+      <div className={classes.text}>
+        Locale:
+      </div>
+      <TextField
+        select
+        value={currentLocale}
+        onChange={onLocaleChange}
+      >
+        <MenuItem value="fr-FR">Le français (fr-FR)</MenuItem>
+        <MenuItem value="de-GR">Deutsch (de-GR)</MenuItem>
+        <MenuItem value="en-US">American (en-US)</MenuItem>
```

Comment:
It's typically called 'English (United States)'

Reply:
Same about other languages - Français (France) and Deutsch (Deutschland)
so we won't need to specify culture codes